src/fake_bpy_module/generator/translator.py

```diff
@@ -57,6 +57,23 @@ def depart_section(self, _: nodes.section) -> None:
         self.doc_writer.addln("--------------------")
         self.doc_writer.new_line()
 
+    def visit_substitution_definition(
+        self, node: nodes.substitution_definition
+    ) -> None:
+        name = node.attributes["names"][0]
+        self.doc_writer.add(f"SUBSTITUTION REPLACEMENT '{name}':")
+        self.doc_writer.new_line()
+        print(
```

Comment:
Please use `output_log` instead of `print` function to output log.

https://github.com/nutti/fake-bpy-module/blob/bf7ccb339e79f14bf8fc9001aff7514c46fa75ca/src/fake_bpy_module/utils.py#L28